desktop/storeService.ts

```diff
@@ -1,11 +1,55 @@
-const Store = require('electron-store');
+import Store from 'electron-store';
+import { TypedSchemaStore } from '../shared/types';
 
 class StoreService {
-  static store: typeof Store;
+  public static store: Store<TypedSchemaStore>;
 
   static init() {
     if (!StoreService.store) {
-      StoreService.store = new Store();
+      StoreService.store = new Store<TypedSchemaStore>({
+        accessPropertiesByDotNotation: true,
+        schema: {
+          netSettings: {
+            type: 'object',
+            properties: {}
+          },
+          accounts: {
+            type: 'object',
+            properties: {}
+          },
+          isAutoStartEnabled: {
+            type: 'boolean',
+            properties: {}
+          },
+          localNode: {
+            type: 'boolean',
+            properties: {}
+          },
+          userSettings: {
+            type: 'object',
+            properties: {}
+          }
+        },
+        defaults: {
+          accounts: {},
+          userSettings: {
+            darkMode: 'system'
```

Comment:
Why do we need a "third" state of boolean as a magic `system` string?
Can we just use `nativeTheme.shouldUseDarkColors` in defaults?

And please, do not set defaults for `netSettings` (and I'm not sure about `localNode`).

Reply:
Right